src/qtcreatorlibrary.pri

```diff
@@ -7,7 +7,7 @@ include(../qtcreator.pri)
 # use precompiled header for libraries by default
 isEmpty(PRECOMPILED_HEADER):PRECOMPILED_HEADER = $$PWD/shared/qtcreator_pch.h
 
-win32|os2 {
+win32 {
```

Comment:
Why is OS/2 removed?

Reply:
because dlltarget seems not be supported for os2, I see it only in win32 qmake code.